strings/phobia.json

```diff
@@ -213,5 +213,59 @@
 		"as (unknown)",
 		"unknown",
 		"stranger danger"
+	],
+
+"birds": [
+		"when are we adding vox",
+		"when you send me the sprites",
+		"or just link me to whichever server repo has them",
+		"i will port them right now",
+		"birdemic",
+		"bird",
+		"beak",
+		"poly",
+		"wing",
+		"wings",
+		"claw",
+		"peck",
+		"pecker",
+		"cracker",
+		"kiki yaya",
+		"ki ki ya ya",
+		"vox",
+		"flaps their wings",
+		"flaps his wings",
+		"flaps her wings",
+		"flaps their wings angrily",
+		"flaps his wings angrily",
+		"flaps her wings angrily",
```

Comment:
emotes are not processed, only spoken chat

Reply:
This makes me sad.